src/style.css

```diff
@@ -0,0 +1,100 @@
+
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+:root {
+  --border-color: #e1e7ed;
+  --border-radius: 4px;
+  --font-family: "Avenir", serif;
+  --color: #3d4e61;
+}
+
+.searchBar {
+  position: relative;
```

Comment:
Consider using more descriptive class names instead of `searchBar`. For example, `searchContainer` or `searchWrapper`. 🤔